scripts/1.3.12/README.md

```diff
@@ -1,8 +1,14 @@
-# Scripts for v1.3.12
+# Scripts for v1.3.12 (released as v1.4.0)
+
+### Downgrade
+
+Required to downgrade from 1.3.12 (released as v1.4.0) to any older version. The script needs to be executed after closing Neo4j Desktop 1.3.12 (released as v1.4.0) and before starting the older version.
+
+This script is needed for projects to be discovered correctly by older versions of Desktop. Without it, you won't be able to startup Desktop (as it won't find the project you last opened). The script is removing some symbolic links and metadata used by Desktop, no data is modified by it.
 
 ### Upgrade (only for Canary)
 
-Required to upgrade the Canary from any version above 1.3.4 to 1.3.12.
+Required to upgrade the Canary from any version above 1.3.4 to 1.3.12 (released as v1.4.0).
```

Comment:
Looking again, Canary seems to have only been released as `1.3.12-dev.x`?

Reply:
Yes, correct. Each canary gets the next version in line and ends with the build number.

Reply:
Then this change here needs to be removed/reworded anyway.

As for the upgrade script, my understanding is that since we will now have separate environments for Desktop, Canary and Development, a Canary upgrade would now have a `relateDataPath` env config property which would not be pointing to the correct place (since we don't actually move any data). This would need to be removed by the script to default to the global Relate data dir as before and pick up DBMSs correctly. This would not be an issue with fresh installs of latest Canary.

Reply:
The upgrade script will be needed for the next canary versions if people didn't upgrade this time, so I think it's best to keep it.